### ISSUE TITLE
Unable to upgrade from K2 2.7.0

### ISSUE THREAD

**Issue by Simon-Davies** [joomlaworks/k2-v3-dev-build#259]
Warning

Unknown column 'asset_id' in 'field list' SQL=INSERT INTO `j40jm_k2_categories` (`id`, `asset_id`, `parent_id`, `lft`, `rgt`, `path`, `level`, `title`, `alias`, `state`, `access`, `description`, `image`, `extra_fields`, `template`, `inheritance`, `created`, `created_by`, `created_by_alias`, `modified`, `modified_by`, `checked_out`, `checked_out_time`, `metadata`, `plugins`, `params`, `language`) VALUES (1, 0, 0, 0, 1, '', 0, 'ROOT', 'root', 0, 1, '', '', '', '', 0, '0000-00-00 00:00:00', 0, '', '0000-00-00 00:00:00', 0, 0, '0000-00-00 00:00:00', '', '', '', '');
Error

Error installing component

**Comment by kavadas** [comment]
Hi,

It seems that somehow you got the old database table structure. How did you tried to upgrade? Provide detailed steps. Keep in mind that the latest stable K2 version currently is 2.6.9.

**Comment by Simon-Davies** [comment]
I tried to upgrade from K2 2.6.9 to 3.0 but it failed with the same error. So I then upgraded to the dev build 2.7.0 and tried again. It then failed again with the same error. I downloaded the master.zip from github and tried to install it through the Joomla backend just how I've installed every other extension successfully. A few months back K2 3.0 was installing fine but I was unable to edit items so I did a database and website files restore to the previous day
